src/default-theme/variables.css

```diff
@@ -0,0 +1,72 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+
+:root {
+
+  /* Borders */
+
+  --jp-borderWidth: 1px;
```

Comment:
Either use camelCase or dashes, but please don't use them both. Below, you have `font-color`, `fontSize`, etc.